packages/components/text/src/text.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <render />
+</template>
```

Comment:
IMO, It feels strange to use template and render function together.

Reply:
Ok, I'll rewrite the template using component tag.